### PR TITLE
LATAM : Add channels + ALL : Cleaning

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Astra streams, following Dum's MTV 80's addition.

### CODE REVIEW

**Comment by Dum4G** [comment]
Please finish cleaning at least in a bot branch. We have 3 cleaning branches atm, none is finished.

**Comment by Dum4G** [comment]
Please check the US playlist for plex wurl links. AMC Weddings from them is not working for me

File: channels/es.m3u [hunk 1]
```diff
@@ -83,8 +83,10 @@ https://cls.alcarria.tv/live/alcarriatv-livestream.m3u8
 https://cartv.streaming.aranova.es/hls/live/aragonradio_aragonradio1.m3u8
 #EXTINF:-1 tvg-id="AragonTV.es" tvg-name="Aragón TV" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d4/Logo_aragon_tv_2016.png" group-title="",Aragón TV (720p)
 https://cartv.streaming.aranova.es/hls/live/aragontv_canal1.m3u8
-#EXTINF:-1 tvg-id="AtresSeriesInternacional.es" tvg-name="Atres Series Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atres Series Internacional
+#EXTINF:-1 tvg-id="AtreseriesInternacional.es" tvg-name="Atreseries Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atreseries Internacional
```
Comment (Dum4G) [inline channels/es.m3u:86]: Internacional but Spanish language and Spain as a country? I'm confused

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/es.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/es.m3u:86]: Atreseries Internacional is a subsidiary channel of the Atresmedia Group, which is Spanish-based. That's why it's in the Spanish section.

https://www.lyngsat.com/tvchannels/es/Atreseries.html

Comment (LaneSh4d0w) [reply channels/es.m3u:86]: To be honest, the "Internacional" mention is only from the hashtag they put over their programs "#A3SeriesInt"

**Comment by LaneSh4d0w** [comment]
Reason for the cleaning in US : 

- XUMO links that could be shortened
- Chunklist links and dupes while a playlist file existed.
- I kept only one channel (MTV Nordic) with his dupe 3500.m3u8, to see if the master was geo-locked or not.

**Comment by LaneSh4d0w** [comment]
Will be good for now. New PR soon for more channels, but this big US wash-up will make some good to the playlists.

File: channels/us.m3u [hunk 2]
```diff
@@ -1874,12 +1852,6 @@ https://dai.google.com/linear/hls/event/EXltT2IOQvCIn8v23_15ow/master.m3u8
 https://dai.google.com/linear/hls/event/knBsxnquSYqFXTP_UzcGgw/master.m3u8
 #EXTINF:-1 tvg-id="IDG.us" tvg-name="IDG" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/M0omWCW.jpg" group-title="News",IDG
 https://a.jsrdn.com/broadcast/529a360c04/+0000/c.m3u8
-#EXTINF:-1 tvg-id="IDG.us" tvg-name="IDG" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/M0omWCW.jpg" group-title="News",IDG
-https://a.jsrdn.com/broadcast/529a360c04/+0000/high/c.m3u8
-#EXTINF:-1 tvg-id="dbd25a" tvg-name="IFC_Films_Picks" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/XJaSjhm.png" group-title="",IFC Films Picks
-https://amc-ifc-films-picks-1.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="IFCFilmsPicks.us" tvg-name="IFC Films Picks" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/XJaSjhm.png" group-title="Movies",IFC Films Picks
-https://amc-ifc-films-picks-1.vizio.wurl.com/manifest/1200.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:1882]: works

Comment (LaneSh4d0w) [reply channels/us.m3u:1882]: Dupe of "AMC Films Picks by IFC"

**Comment by Dum4G** [comment]
Both MTV Nordics do not work for me, not even with VPN

File: channels/us.m3u [hunk 3]
```diff
@@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
 https://thegateway.app/YouToo/CueTones/playlist.m3u8
 #EXTINF:-1 tvg-id="YUYUTV.us" tvg-name="YUYU TV" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/Bbvtcd7.png" group-title="",YUYU TV
 https://yuyu-samsung.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living (720p)
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r/index.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:6652]: works

File: channels/us.m3u [hunk 4]
```diff
@@ -6648,10 +6438,6 @@ https://gideommd.mmdlive.lldns.net/gideommd/d5b6a753f86e4e51b8e15c7d23887a4f/pla
...
-#EXTINF:-1 tvg-id="ZLiving.us" tvg-name="Z Living" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/c3Z5i2L.png" group-title="Classic",Z Living
-https://y5w8j4a9.ssl.hwcdn.net/zlivingusa_r_prod/index.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:6654]: works

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by LaneSh4d0w** [comment]
Noted, i'm gonna see if I can get a working link.

File: channels/us.m3u (hunk 4 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:6654]: Already available on the Indian playlist.

File: channels/us.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/us.m3u:6652]: Already available on the Indian playlist.

Comment (Dum4G) [reply channels/us.m3u:6652]: It's wrong then
https://en.wikipedia.org/wiki/Z_Living

Comment (LaneSh4d0w) [reply channels/us.m3u:6652]: https://www.lyngsat.com/tvchannels/in/Z-Living-USA.html

Comment (Dum4G) [reply channels/us.m3u:6652]: they're wrong :)

**Review by Dum4G — APPROVED** [review]

**Pull request merged**